model/driver/src/icon4py/model/driver/initialization_utils.py

```diff
@@ -559,3 +570,33 @@ def configure_logging(
     console_handler.setFormatter(formatter)
     console_handler.setLevel(logging_level)
     logging.getLogger("").addHandler(console_handler)
+
+
+def create_grid_info(
```

Comment:
I think it s easiest to read this off the gridfile directly here. Keeping in mind that we really need to rework this driver. All grid file handling should eventually be done in the GridManager, but as we still use the serialized data here, we should keep constructing the grid from it as in some places ordering of local dimension is not identical.